src/lib/apis/fetch.ts

```diff
@@ -17,7 +17,7 @@ interface URLAndRequestBodyParams extends RequestBodyParams {
 export const constructUrlAndParams = (method, url): URLAndRequestBodyParams => {
   const opts: RequestBodyParams = {}
 
-  if (method === "PUT" || method === "POST") {
+  if (method === "PUT" || method === "POST" || method === "DELETE") {
```

Comment:
so that `password` param is not exposed in url query string. see [thread](https://artsy.slack.com/archives/CP9P4KR35/p1652393249677269).

Reply:
exercised the DELETE via local MP pointing to staging Gravity. confirmed params are _not_ logged by Gravity:
```
May 16 13:39:52 gravity-web-6c4d95d467-d4xm2 k8s_gravity-web_gravity-web-6c4d95d467-d4xm2_def Started DELETE "/api/v1/me" for 172.70.230.33 at 2022-05-16 13:39:52 +0000
May 16 13:39:52 gravity-web-6c4d95d467-d4xm2 k8s_gravity-web_gravity-web-6c4d95d467-d4xm2_def  [api] (13) {"method":"DELETE","path":"/api/v1/me","query":"","request_id":"a9deedd0-d51d-11ec-a376-d97a82de5975","user_id":"628253752a2627000d52a1ef","client_id":"56451872ebad64110e000000"}
```